### PR TITLE
Initial Thermostable TdT and ssDNA to dsDNA plans, #36

### PR DESCRIPTION
**Pull request by TunaW**

### CODE REVIEW

**Comment by dperezhi** [comment]
For the "ssDNA to dsDNA Experimental Design" part, the Context and Scope section needs a reference. If you want, try using the Bibliography format Lucy set up, it's pretty nice; to see how it is set up check the Microfluidics index.md file as well as the bibliography.bib file (within the src folder).

**Comment by lhao03** [comment]
I have to add the necessary packages to the replit setup; so right now the bibliography.bib doesn't work on replit (but does work on Codespaces)

**Comment by lhao03** [comment]
But for now, like Diego mentioned, you can add references through bibliography.bib, they just won't render correctly

**Comment by lhao03** [comment]
I merged main/resolved some merge conflicts so you can now add sources to bibliography.bib. Make sure to `git pull` before adding new commits since I pushed some changes. Thanks!

**Review by dperezhi — COMMENTED** [review]
ssDNA to dsDNA Experimental Design, context and scope, needs a reference

**Review by lhao03 — CHANGES_REQUESTED** [review]

**Review by BurakMOzkan — CHANGES_REQUESTED** [review]
Some grammar mistakes. I recommend having key words and then a dash to further explain in detail the addition. Great work otherwise!

File: src/wet-lab/design/tdt-experiment.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+**Solid Phase (TBD)**
+
+## How long will this take?
+
+This depends heavily on how successfully each experiment goes. The estimation is around 1 month.
+
+## Reference:
+
+Liang, X., Kuhn, H., & Frank-Kamenetskii, M. D. (2006). Monitoring single-stranded DNA secondary structure formation by determining the topological state of DNA catenanes. Biophysical Journal, 90(8), 2877–2889. https://doi.org/10.1529/biophysj.105.074104
```
[the remark this answers is not in the document]
Comment (lhao03) [reply src/wet-lab/design/tdt-experiment.md:109]: https://github.com/UBC-iGEM/internal-wiki-2023-24/blob/2073b9308f77212a04f615918a17f7db0e55ded0/src/bibliography.bib#L19-L30

Comment (lhao03) [inline src/wet-lab/design/tdt-experiment.md:109]: https://github.com/UBC-iGEM/internal-wiki-2023-24/blob/073f715ae595a904d7102099ebbb248b28afde57/src/bibliography.bib#L167-L180

File: src/bibliography.bib [hunk 2]
```diff
@@ -39,3 +39,13 @@ @article{outlook_microfludics
   volume          = 3,
   year            = 2022
 }
+
+@Comment TdT Experiment 
+@article{TdT background,
```
Comment (lhao03) [inline src/bibliography.bib:44]:
```suggestion
@article{TdT_background,
```

Sorry need dash here, but not for `@Comment TdT Experiment`

File: src/wet-lab/design/tdt-experiment.md (hunk 1 above)
Comment (lhao03) [inline src/wet-lab/design/tdt-experiment.md:109]:
```suggestion
[@TdT_background]
```

**Review by lhao03 — APPROVED** [review]
I'll merge this in for now, but open an issue stating the changes requested, since the TdT plans are changing, so everyone can start reading about the wet lab plans.

**Pull request merged**